go.mod

```diff
@@ -52,7 +52,6 @@ replace (
 	github.com/decred/dcrd/dcrec => ./dcrec
 	github.com/decred/dcrd/dcrec/edwards => ./dcrec/edwards
 	github.com/decred/dcrd/dcrec/secp256k1 => ./dcrec/secp256k1
-	github.com/decred/dcrd/dcrjson => ./dcrjson
```

Comment:
No change in the requires, so the main module is still using v1.  Probably intentional, but just want to confirm.

Reply:
Correct.  If the override is left in, it will try to build with the code in the repository which is already v2 and thus fail with missing types.

Reply:
I should note that I plan to put the override back in once everything is updated to use the new v2 module and the associated types from dcrwallet are available.

Reply:
want to update the required module to latest released version?  There are some additional changes besides just the walletinfo cointype change:

```
jrick@dev:~/src/dcrd/dcrjson$ git diff dcrjson/v1.1.0 -- .                                                                                                                
diff --git a/dcrjson/register.go b/dcrjson/register.go
index 0c1a81a..ab3a489 100644
--- a/dcrjson/register.go
+++ b/dcrjson/register.go
@@ -288,6 +288,6 @@ func RegisteredCmdMethods() []string {
                methods = append(methods, k)
        }
 
-       sort.Sort(sort.StringSlice(methods))
+       sort.Strings(methods)
        return methods
 }
diff --git a/dcrjson/register_test.go b/dcrjson/register_test.go
index 909a5bd..57a5b2f 100644
--- a/dcrjson/register_test.go
+++ b/dcrjson/register_test.go
@@ -254,7 +254,7 @@ func TestRegisteredCmdMethods(t *testing.T) {
        // Ensure the returned methods are sorted.
        sortedMethods := make([]string, len(methods))
        copy(sortedMethods, methods)
-       sort.Sort(sort.StringSlice(sortedMethods))
+       sort.Strings(sortedMethods)
        if !reflect.DeepEqual(sortedMethods, methods) {
                t.Fatal("RegisteredCmdMethods: methods are not sorted")
        }
diff --git a/dcrjson/walletsvrresults.go b/dcrjson/walletsvrresults.go
index 4898fa6..44f8409 100644
--- a/dcrjson/walletsvrresults.go
+++ b/dcrjson/walletsvrresults.go
@@ -412,11 +412,11 @@ type VerifySeedResult struct {
        CoinType uint32 `json:"cointype"`
 }
 
-// WalletInfoResult models the data returned from the walletinfo
-// command.
+// WalletInfoResult models the data returned from the walletinfo command.
 type WalletInfoResult struct {
        DaemonConnected  bool    `json:"daemonconnected"`
        Unlocked         bool    `json:"unlocked"`
+       CoinType         uint32  `json:"cointype,omitempty"`
        TxFee            float64 `json:"txfee"`
        TicketFee        float64 `json:"ticketfee"`
        TicketPurchasing bool    `json:"ticketpurchasing"`
```

Reply:
Done.